CharacterCreation/04_05_Long_Rifles_&_DMRs.txt

```diff
@@ -1,71 +1,83 @@
-On all engagements farther than 200m, you must take 2 Actions to adjust							
-your scope for windage and elevation between targets of differing distance.							
-Minimum 5 Strength. Bolt Action: take 1 action to fire, 1 action to chamber a new round. 							
-Reload time 3 Actions.		
+    The kings of rifles, long rifles are the longest ranged and hardest hitting 
+rifles available. On all engagements farther than 200m, you must take 2 Actions 
+to adjust your scope for windage and elevation between targets of differing 
+distance.							
 
-- All Long Rifles/DMRs have "Suppression Level - light" (unless otherwise noted).
+	Minimum 5 Strength. Bolt Action: take 1 action to fire, 1 action to chamber 
+a new round. Reload time 3 Actions.		
 
+	All Long Rifles/DMRs have "Suppression Level - light" (unless otherwise 
+noted).
 
 =====	LEVEL 1 REQUIREMENT WEAPONS	=====
 
-----Anvil Arms SR-150	$1559	----
-Damage:	115 Ballistic	
-Ranged Miss Chance:		
-	3 out to 20m	
-	4 out to 100m	
-	5 out to 170m	
+== Anvil Arms SR-150	$1559	==
+Damage:	115 Ballistic
 Mag Size:		5 rounds of 7.65x54mm. $10 per magazine
 Reload DC:		35
 Fire Modes:		Bolt Action
 Armor Piercing Level:	3	
 Reflex Modifier:	-25	
 Movement Speed Penalty:	-1m	
-Notes/Description:	Starts with 'Basic 2x Scope attached': +10m minimum range	
+
+	Starts with 'Basic 2x Scope attached': +10m minimum range
+
++---------+-----+------+------+
+| Range   | 20m | 100m | 170m |
++---------+-----+------+------+
+| To Miss |  3  |   4  |   5  |
++---------+-----+------+------+
 		
-----ECSC Mk12 Mod 0 	$2159	----
+== ECSC Mk12 Mod 0 	$2159	==
 Damage:	105 Ballistic
-Ranged Miss Chance:		
-	3 out to 20m	
-	4 out to 65m	
-	5 out to 110m	
 Mag Size:		20 rounds of 5.77x55mm. $53 per magazine
 Reload DC:		29
 Fire Modes:		Semi
 Armor Piercing Level:	2	
 Reflex Modifier:	-15	
 Suppression Level:	Medium	
 Movement Speed Penalty:	-0.5m	
-Notes/Description:	Starts with 'Basic 2x Scope attached': +10m minimum range	
+
+	Starts with 'Basic 2x Scope attached': +10m minimum range
+
++---------+-----+------+------+
+| Range   | 20m |  65m | 110m |
++---------+-----+------+------+
+| To Miss |  3  |   4  |   5  |
++---------+-----+------+------+
 
 =====	LEVEL 5 REQUIREMENT WEAPONS	=====
 
-----ECSC Mk25 DMR	$2877	----
+== ECSC Mk25 DMR	$2877	==
 Damage:	110 Ballistic
-Ranged Miss Chance:		
-	2 out to 20m	
-	3 out to 75m	
-	4 out to 130m	
 Mag Size:		10 rounds of 7.65x51mm. $28 per magazine
 Reload DC:		29
 Fire Modes:		Semi
 Armor Piercing Level:	3	
 Reflex Modifier:	-20	
 Suppression Level:	Medium	
 Movement Speed Penalty:	-0.5m	
-Notes/Description:	Starts with 'Basic 2x Scope attached': +10m minimum range	
+
+	Starts with 'Basic 2x Scope attached': +10m minimum range	
+
++---------+-----+------+------+
+| Range   | 20m |  75m | 130m |
++---------+-----+------+------+
+| To Miss |  2  |   3  |   4  |
++---------+-----+------+------+
 		
-----ECSC PRS 	$2303	----
+== ECSC PRS 	$2303	==

```

Comment:
Old Miss Chance Format persists on this weapon, and it doesn't seem to have the new table.